api/ruvnl-morning.ts

```diff
@@ -69,6 +71,13 @@ const checkEmailResponse = (message: string, source: "Wind" | "Solar", resendRes
   return message;
 };
 
+const getTomorrowDateString: () => string = () => {
+  const today = new Date();
+  const tomorrow = new Date();
+  tomorrow.setDate(today.getDate() + 1);
+  return tomorrow.toLocaleDateString();
```

Comment:
Could you get it to return YYYY-MM-DD, so there is no American/English confusion

Reply:
Currently it is "05/04/2012" which could get confusion, depending which data format you use.

Reply:
https://en.wikipedia.org/wiki/ISO_8601, I think YYYY-MM-DD is most common

Reply:
Yep fair, I was using Locale as that's what I'm used to using and will display in the user's browser format – but we're not in the browser, so it'll always give the Vercel set region which is UK/US depending 😅 

Have changed to ISO as per suggestion